<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
padding: 4px;
border-radius: 4px;
border: 1px solid rgba(55, 55, 55);
font-weight: bold;
/*
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
*/
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.01 rgba(120, 120, 120),
stop: 0.05 rgba(150, 150, 150),
stop: 0.5 rgba(120, 120, 120),
stop: 0.52 rgba(80, 80, 80),
stop: 0.9 rgba(50, 50, 50),
stop: 1.0 rgba(50, 50, 50));
color: rgba(225, 225, 225);
}

QPushButton:pressed {
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(25, 25, 25),
stop: 0.15 rgba(75, 75, 75),
stop: 0.85 rgba(25, 25, 25),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}

QPushButton:disabled {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(50, 50, 50),
stop: 0.05 rgba(100, 100, 100),
stop: 0.15 rgba(150, 150, 150),
stop: 0.85 rgba(100, 100, 100),
stop: 1.0 rgba(50, 50, 50));
}

QLabel {
color: rgba(35, 35, 35);
font-weight: normal;
background: transparent;
border: none;
}

QCheckBox {
background: transparent;
border: none;
}
/**
QPushButton {
border-radius: 4px;
border: 1px solid rgba(75, 75, 75);
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
color: rgba(225, 225, 225);
}
/**
QPushButton:pressed {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(0, 0, 0),
stop: 0.05 rgba(50, 50, 50),
stop: 0.15 rgba(100, 100, 100),
stop: 0.85 rgba(50, 50, 50),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}**/

#Widget {
	background: rgba(200, 200, 200);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 2px;
border-bottom-right-radius: 2px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop: 0.0 rgba(125, 125, 125),
stop: 0.02 rgba(200, 200, 200),
stop: 0.98 rgba(200, 200, 200),
stop: 1.0 rgba(125, 125, 125));
}

QLineEdit {
border: 1px solid rgba(175, 175, 175);
background: rgba(255, 255, 255);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-bottom-left-radius: 4px;
}

QGroupBox {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin-top: 1ex;
}

QGridLayout {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin: 10px;
}

#simulatorButton {
width: 100px;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_r">
       <property name="text">
        <string>Rock</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_p">
       <property name="text">
        <string>Paper</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_s">
       <property name="text">
        <string>Scissors</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Play!</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="statusLabel">
       <property name="text">
        <string>Nothing yet!</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Enemy chose:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Nothing Yet!</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Wins</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="winLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Draws</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="drawLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Loses</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="loseLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear statistics</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
